### DISCUSSION TITLE
美大學研究 3D 列印技術，可印出 OLED 面板

### DISCUSSION THREAD

**Discussion by blackline0911** [1101-GS4719-C/Python-Community-GS4719-C#637] [General]
https://technews.tw/2022/01/12/breakthrough-could-help-you-3d-print-oled-screens-at-home/

文中發表了一個用3D列印出OLED螢幕面板的技術，OLED面板是相對我們常用的LCD面板，他的發光來源不是LED，而是發光有機材料，具有低耗電的特性。雖然文中發表的面板只能製作出64像素，但是OLED面板未來的發展相信一定會非常看好

**Comment by stan890425** [comment]
3D列印通常是以有機材料為主，剛好可以跟OLED搭配，可以有很好的匹配度。

**Comment by JSHT** [comment]
目前主流的方式使用蒸鍍(代表：三星)、噴墨打印(代表：華星)，都已經發展到一定程度，可以說是技術很成熟了，可以看[這部影片的介紹](https://www.youtube.com/watch?v=6QpePfw3Ip0)。3D列印的方式既新奇，未來提出家用版的方案可行性也蠻高的。
